bin/reload-upstream

```diff
@@ -58,10 +64,10 @@ function fetchdocs () {
 # finally, apply our references
 #
 
-fetchdocs docs/boshrelease https://github.com/logsearch/logsearch-boshrelease.git docs
-fetchdocs docs/shipper-boshrelease https://github.com/logsearch/logsearch-shipper-boshrelease.git docs
-fetchdocs docs/filters-common https://github.com/logsearch/logsearch-filters-common.git docs
-fetchdocs docs/workspace https://github.com/logsearch/workspace.git docs
+fetchdocs docs/boshrelease https://github.com/logsearch/logsearch-boshrelease.git docs getting-started-with-logsearch-workspace
```

Comment:
The reason I went down this route was that I expected to be making changes in multiple sub-projects, and wanted to be able to "publish" all of them in a single place for people to review.

By extending `reload-upstream` in this manor I can piggy back on the "every logsearch-website PR gets its own "PR" website" feature we already have, and get a nice simple "url" for people to review a collection of changes.  So, from that perspective its a useful (if seldom used) feature.

I'd like to keep it, but will default the branch to `master`.

The achilles heal with this technique is that I need to remember to change the reference back to `master` once the upstream docs get merged.